src/test/components/ProfileThreadTracingMarkerOverview.test.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
a lot of this file was copied over from TimelineMarkers.test.js; please tell me if you'd like that I move some utility functions to a shared file.